src/Spd.Presentation.Licensing/Controllers/BizPortalUserController.cs

```diff
@@ -85,4 +85,30 @@ public async Task<BizPortalUserListResponse> GetBizPortalUserList([FromRoute] Gu
     {
         return await _mediator.Send(new BizPortalUserListQuery(bizId));
     }
+
+    /// <summary>
+    /// Get Business Portal User
+    /// </summary>
+    /// <param name="bizId"></param>
+    /// <returns></returns>
+    [Authorize(Policy = "OnlyBCeID", Roles = "PrimaryManager,Manager")]
+    //TODO: ask about the route address, should it be '/business' instead of 'bizs'?
```

Comment:
Yes

Reply:
It seems everyone wants to change name from biz to business. For me, I think business is too long. But if you guys want, we can change all "biz" to business. Up to you guys.

Reply:
I just noticed every other routes are lik api/business. I'm Ok with either. @peggy-quartech